channels/ir.m3u

```diff
@@ -127,6 +141,8 @@ http://d2e40kvaojifd6.cloudfront.net/stream/maxx/playlist.m3u8
 https://d2e40kvaojifd6.cloudfront.net/stream/mifa/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="Mihan TV" tvg-language="Persian" tvg-logo="http://mihantv.com/wp-content/themes/mihantv/images/logo.png" group-title="News",Mihan TV
 https://iptv.mihantv.com/live/playlist1/index.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="Modern Economy" tvg-language="Persian" tvg-logo="https://www.gemonline.tv/Assets/channels-box/modern_economy.png?13" group-title="Entertainment",Modern Economy
+https://d2e40kvaojifd6.cloudfront.net/stream/modern_enonomy/playlist_960x540_2000k.m3u8
```

Comment:
Link do not work.

Reply:
I just checked with https://castr.io/hlsplayer and it is working

Reply:
This is what I see on https://castr.io/hlsplayer
<img width="983" alt="Screenshot 2020-09-12 at 14 46 31" src="https://user-images.githubusercontent.com/7253922/92994864-ee9b8580-f506-11ea-86e5-4cde9d1fa720.png">

Reply:
I am not sure what is going on but try this stream which is lower in quality.
https://d2e40kvaojifd6.cloudfront.net/stream/modern_economy/playlist_640x360_800k.m3u8

Reply:
Same thing.

Reply:
I just checked with VLC and https://castr.io/hlsplayer and both are playing.

Reply:
This is what I see in VLC:
<img width="317" alt="Screenshot 2020-09-12 at 20 38 41" src="https://user-images.githubusercontent.com/7253922/93001492-09d2b900-f538-11ea-9b25-af4474df6ccd.png">

Reply:
![vlcsnap-2020-09-12-13h50m42s894](https://user-images.githubusercontent.com/61093878/93001809-ad53a780-f4ff-11ea-8312-de30779f0c48.png)
This is a snapshot of my VLC player. I also loaded the m3u in firestick and plays in there with tivi mate

Reply:
Then I assume the broadcast is geoblocked.

Reply:
I don't think so because all the Gem channels are from the same server.

Reply:
Working on Singapore IP

<img width="640" alt="Screenshot 2020-09-14 at 2 10 06 PM" src="https://user-images.githubusercontent.com/22504259/93049649-03f1ea80-f694-11ea-81f2-415041413546.png">

Reply:
@data14a But the fact is that all channels from this server give out an error.

Reply:
I just checked the streams from this server using VLC in windows 7 and also in windows 10. Checked with tivimate on firestick and with https://www.jwplayer.com/developers/stream-tester/. They all play fine so the error might be on your side.

Reply:
Maybe you are right and these links do not work only for me.

Reply:
I am not sure what to do at this point. If you are not going to approve this change you can close the request.